### PR TITLE
Monica/ui fixes

### PR DESCRIPTION
**Pull request by myo3**

### CODE REVIEW

File: TCAT/Model/Route.swift [hunk 1]
```diff
@@ -144,14 +147,17 @@ class Route: NSObject, JSONDecodable {
...
-     *      remove the first routeSummaryObject
+     *      remove the first routeSummaryObject and update the time
      *  Else (the first routeSummaryObject is a bus stop), so simply update the name to the name the user searched for
      */
     func updateStartingDestination(_ place: Place) {
         if let firstRouteSummaryObject = routeSummary.first {
             if firstRouteSummaryObject.name == "Start" {
                 if (place is PlaceResult || (place is BusStop && place.name == "Current Location")) && routeSummary.count > 2 {
                     routeSummary.remove(at: 0)
+                    departureTime = (routeSummary.first?.time)!
```
Comment (AAAstorga) [inline TCAT/Model/Route.swift:158]: Is this force unwrap ok? Will it always always work?

**Review by AAAstorga — APPROVED** [review]
I left a comment, but besides that it looks great!

File: TCAT/Model/Route.swift (hunk 1 above)
Comment (myo3) [reply TCAT/Model/Route.swift:158]: yes b/c i check `routeSummary.count > 2` in if statement so `routeSummary.first?` will never be nil and `time` is not an optional property in `RouteSummaryObject`

Comment (myo3) [reply TCAT/Model/Route.swift:158]: but agree this not good style >.<

**Pull request merged**